modules/nw-autoscaling-ingress-controller.adoc

```diff
@@ -27,12 +30,12 @@ $ oc create serviceaccount thanos && oc describe serviceaccount thanos
 [source,terminal]
 ----
 Name:                thanos
-Namespace:           openshift-ingress-operator
+Namespace:           default
 Labels:              <none>
-Annotations:         <none>
-Image pull secrets:  thanos-dockercfg-b4l9s
-Mountable secrets:   thanos-dockercfg-b4l9s
-Tokens:              thanos-token-c422q
+Annotations:         openshift.io/internal-registry-pull-secret-ref: thanos-dockercfg-kfvf2
```

Comment:
This annotation appears to be added in 4.16, so this is accurate for 4.16 and 4.17.

But if you plan to backport to anything earlier than 4.16, I would remove this annotation in the < 4.16 backports, since we should avoid implying it should be there, just to avoid any confusion.

Reply:
I'll ask Ishmam if the changes should stem back to earlier releases.